smartsupp.php

```diff
@@ -312,20 +334,27 @@ public function hookHeader()
     public function hookBackOfficeHeader()
     {
         $js = '';
+
         if (strcmp(Tools::getValue('configure'), $this->name) === 0) {
             $path = $this->_path;
-            if (version_compare(_PS_VERSION_, '1.6', '>=') == true) {
-                $this->context->controller->addJquery();
-                $this->context->controller->addJs($path . 'views/js/smartsupp.js');
-                $this->context->controller->addCSS($path . 'views/css/smartsupp.css');
-                if (version_compare(_PS_VERSION_, '1.6', '<') == true) {
-                    $this->context->controller->addCSS($path . 'views/css/smartsupp-nobootstrap.css');
-                }
-            } else {
-                $js .= '<script type="text/javascript" src="'.$path.'views/js/smartsupp.js"></script>';
-                $js .= '<link rel="stylesheet" href="'.$path.'views/css/smartsupp.css" type="text/css" />';
-                $js .= '<link rel="stylesheet" href="'.$path.'views/css/smartsupp-nobootstrap.css" type="text/css" />';
-            }
+            $js .= '<script type="text/javascript" src="' . $path . 'views/js/smartsupp.js"></script>';
+            $js .= '<link rel="stylesheet" href="' . $path . 'views/css/smartsupp.css" type="text/css" />';
+            $js .= '<link rel="stylesheet" href="' . $path . 'views/css/smartsupp-nobootstrap.css" type="text/css" />';
+        }
+
+        return $js;
```

Comment:
would be much better to return tpl file with this code

Reply:
This work is not in current scope, there is many html code inside project and should be done seperatly